docs/docs/docs/developer-resources/testing.md

```
@@ -184,6 +185,49 @@
+
+**NOTE:** This applies only to Talawa API developers.
+
+Sometimes you may want to start all over again from scratch. These steps will reset your development postgres database.
+
+### Using the CLI (Dev Containers)
+
+This applies to users running Talawa API in dev containers.
+
+1. Once the server is running, open a new terminal session.
+2. Open a bash session inside the running container:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)